content/blog/eng/lambda-days-2020.md

```diff
@@ -0,0 +1,91 @@
+---
+authors: ["andreac"]
+comments: true
+date: "2020-04-09"
+draft: false
+share: true
+categories: [English, Conference, Lambda-days, Krakow, Functional programming]
+title: "Lambda-days-2020"
+ita: "Lambda-days-2020"
+
+languageCode: "en-US"
+type: "post"
+---
+
+# Lambda Days 2021
+
+This year, Facile attended for the first time to [Lambda Days](https://www.lambdadays.org/lambdadays2020) Conference in the beautiful polish city of Krakow.
+It has been a two day full immersion, where the organizers have been able to gather speakers from all over the world
```

Comment:
It has been a two-days full immersion with speakers coming from all over the world.